snippets/pyaloha/ccode.py

```diff
@@ -55,7 +55,10 @@ def __init__(self, server_dtime, client_dtime):
 
     @property
     def is_accurate(self):
-        return self.dtime == self.client_creation
+        client_dtime = SerializableDatetime.utcfromtimestamp(
```

Comment:
Why not just simple
```python
def is_accurate(self):
    return self.server_upload == self.client_creation
```
?
No need to convert and compare complex datetime objects, you can compare already present integers.